loki/transformations/parallel/openmp_region.py

```diff
@@ -53,7 +52,7 @@ class RemoveOpenMPRegionTransformer(Transformer):
 
         def visit_PragmaRegion(self, region, **kwargs):
             """
-            Perform the fileterin and removeal of OpenMP pragma regions.
+            Perform the filetering and removal of OpenMP pragma regions.
```

Comment:
```suggestion
            Perform the filtering and removal of OpenMP pragma regions.
```

Reply:
Yes, thanks, will mop this up in the rebase.